Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -77,6 +77,11 @@ void PPCSymbolDB::AddKnownSymbol(u32 startAddr, u32 size, const std::string& nam
     {
       PPCAnalyst::AnalyzeFunction(startAddr, tf, size);
       checksumToFunction[tf.hash] = &(functions[startAddr]);
+
+      tf.function_name = tf.name.substr(0, tf.name.find('('));
```

Comment:
Yeah, it's useless. Returning by non-const value is generally better since the move constructor/assignment can be used in certain circumstances. With const value returns that cannot happen (e.g. the move constructor is `Type(Type&&)`, not `Type(const Type&&)`, so any attempted move with returned const values will always result in a copy; similar things apply to the move assignment operator).

As an aside you may want to make `GetStrippedFunctionName` a local function in the .cpp file, since it's an implementation detail. If it's going to be in the public interface, it probably shouldn't accept arbitrary strings, and take a Symbol argument instead.